aten/src/ATen/ScalarType.h

```diff
@@ -89,6 +89,18 @@ static inline const char * toString(ScalarType t) {
 #undef DEFINE_CASE
 }
 
+static inline size_t elementSize(ScalarType t) {
```

Comment:
Wow, surprised we never had this!

Reply:
The ATen Storage type itself implements this, but we don't have it available here (yet).